components/channel_layout/center_channel/center_channel.jsx

```diff
@@ -64,7 +64,7 @@ export default class CenterChannel extends React.PureComponent {
                             )}
                         />
                         <Route
-                            path={'/:team/:path(channels|messages)/:identifier'}
+                            path={'/:team/:path(channels|messages)/:identifier/:messageId?'}
```

Comment:
Will create another ticket to change url's for permaview in the code along with redirection for older links